Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1028,7 +1028,8 @@ bool TestController::resetStateToConsistentValues(const TestOptions& options, Re
     if (!jscOptions.empty())
         setValue(resetMessageBody, "JSCOptions", jscOptions.c_str());
 
-    WKPagePostMessageToInjectedBundle(TestController::singleton().mainWebView()->page(), toWK("Reset").get(), resetMessageBody.get());
+    if (resetStage == ResetStage::AfterTest)
```

Comment:
Right EWS is green. If it does cause issues, we could pass a flag to the WebContent and then let the WebContent decide what to do before and after the test. The reason I didn't do that are:
1. This would add complexity
2. DumpRenderTree only runs this logic *after* the test, not before
3. The functions on the WebCore side that get called as a result of this are "[something]AfterTest()"
4. EWS shows no flakiness